SuperGiant crashes on new pod resource

Trying to add the following pod to test via the Pods -> New:

```
{
  "kube_name": "senseict",
  "namespace": "default",
  "kind": "Pod",
  "name": "helloworld",
  "resource": {
    "metadata": {
      "labels": {
        "service": "helloworld"
      }
    },
    "spec": {
      "containers": [
        {
          "name": "app",
          "image": "tutum/hello-world"
        }
      ]
    }
  }
}
```

Results in SuperGiant crashing with the following message:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x4a3242]

goroutine 6520 [running]:
panic(0xd6b440, 0xc420018030)
        /usr/local/go/src/runtime/panic.go:500 +0x1a1
github.com/supergiant/supergiant/pkg/core.(*DefaultProvisioner).Provision(0xc420023718, 0xc420764d00, 0x7, 0x0)
        /ext-go/1/src/github.com/supergiant/supergiant/pkg/core/provisioner_default.go:17 +0x62
github.com/supergiant/supergiant/pkg/core.(*KubeResources).Start.func1(0xc420601380, 0x0, 0xc420e87980)
        /ext-go/1/src/github.com/supergiant/supergiant/pkg/core/kube_resources.go:152 +0xc6
github.com/supergiant/supergiant/pkg/core.(*Action).Async.func1(0xc420601380)
        /ext-go/1/src/github.com/supergiant/supergiant/pkg/core/action.go:72 +0x4a
created by github.com/supergiant/supergiant/pkg/core.(*Action).Async
        /ext-go/1/src/github.com/supergiant/supergiant/pkg/core/action.go:93 +0x22d

```

The same issue occurs when attempting to add a new Service in as well.

This is all via the web interface.

--- a/pkg/model/kube_resource.go
+++ b/pkg/model/kube_resource.go
@@ -28,7 +28,7 @@ type KubeResource struct {
 	Name string `json:"name" validate:"nonzero" gorm:"not null;unique_index:kube_namespace_kind_name"`
 
 	// Resource is where the actual Kubernetes definition is stored.
-	Resource     *json.RawMessage `json:"resource" gorm:"-" sg:"store_as_json_in=ResourceJSON,readonly"`
+	Resource     *json.RawMessage `json:"resource" gorm:"-" sg:"store_as_json_in=ResourceJSON"`
 	ResourceJSON []byte           `json:"-"`
 
 	// Started represents whether the resource exists in Kubernetes or not. If it

